radix-engine/tests/abi.rs

```diff
@@ -13,7 +14,8 @@ use transaction::builder::ManifestBuilder;
 #[test]
 fn test_invalid_access_rule_methods() {
     // Arrange
-    let mut test_runner = TestRunner::new(true);
+    let mut store = InMemorySubstateStore::with_bootstrap();
```

Comment:
Why can't we have `TestRunner` owns the store? Looks easier to have a getter method to expose it if needed.

Reply:
I needed to be able to hold a reference in the struct of `TestRunner` to the `store` for the purpose of staged execution. This was the simplest idea I could think of.

Reply:
Is it possible to get a reference through the `TestRunner`? If you can point me to the code, I can also help take a look.

Reply:
In my other PR, I started to use `Rc<dyn ResableSubstateStore>`, so we'll never need a strong reference. I can also do that during merging time. :)

Reply:
You can, but you can't self-reference to the same struct which is what we would need to do.

Reply:
Cool. Let's leave it as-is. It's not a big deal.